### ISSUE TITLE
 Menu component for the web version

### ISSUE THREAD

**Issue by cguedes** [refstudio/refstudio#384]
This is part of #347 

Add a menu for the web that contains the relevant actions we have in the Desktop menu.
Also, we need to capture keyboard events for these menu actions (ex: save).

**Comment by hammer** [comment]
Why do we need a menu for the web version? Aren’t all these commands available through the command palette?

**Comment by danvk** [comment]
Yes, commands would all be available via the command palette. A menu would aid discoverability and consistency across the two platforms. If we're not going to have a menu in the web version then we should probably de-emphasize it in the desktop version, too.

Notion and Google Docs both have menu bars, though Google's is much more prominent/important.

Menu in Notion:

<img width="427" alt="image" src="https://github.com/refstudio/refstudio/assets/98301/9473217e-6072-4789-81bf-9cdb32507107">

Menu in Google Docs:

<img width="474" alt="image" src="https://github.com/refstudio/refstudio/assets/98301/25fa58af-dded-4292-9aa3-606ddbb14a6d">

**Comment by hammer** [comment]
While we're talking about menus, I wanted to note that in [Tauri 2.0](https://github.com/orgs/tauri-apps/projects/11/views/2) there are several big changes to menus, for example https://github.com/tauri-apps/tauri/issues/6617 (already found by @danvk), https://github.com/tauri-apps/tauri/pull/7535, https://github.com/tauri-apps/tauri/issues/4338 (context menus, but related). No action now, but it may impact our implementation strategy if we know a big change is coming.

**Comment by hammer** [comment]
@danvk I can see both sides of this argument. For a desktop app, a menu is quite natural and it's weird not to have it. For a web app, it's becoming increasingly unnatural. For example, [Type](https://type.ai/app.html) and [Wordtune](https://app.wordtune.com/editor) do not have a menu.

Both Slack and Notion use the menu for their desktop app but don't replicate the menu functionality on their web app. Note that Notion has both a top-of-screen menu and the menu you've pointed out that's discoverable under the "•••" in the upper right. 

Overall I think the notion of a web app with a top-of-screen menu is a bit outdated and not necessary. I'm don't hold this position strongly though so if y'all think it's in error please let me know!

**Comment by danvk** [comment]
I'm fine with no menu on the web version. We should, however, make sure that there's some on-screen, discoverable way to complete critical actions in the app. For example Notion auto-saves (so Save / <kbd>Cmd</kbd><kbd>S</kbd> isn't critical) and has an on-screen "+" button to add new documents:
<img width="371" alt="image" src="https://github.com/refstudio/refstudio/assets/98301/25a6af86-bee5-4092-b05f-2ca759115f2e">
